database/sqlite/sqlite_aclk_node.c

```diff
@@ -130,3 +130,25 @@ void sql_build_node_info(struct aclk_database_worker_config *wc, struct aclk_dat
 
     return;
 }
+
+void register_update_node_instance_connection_msg(RRDHOST *host, int cmd)
```

Comment:
Hello @MrZammler ,

I saw in next file that this function is only called inside `#ifdef ENABLE_ACLK`, so I will give a suggestion, instead to have the same `ifdef` inside the function, we could write the whole function inside an `#ifdef ENABLE_ACLK` block.  Or are you thinking to call the same function from more places?

Best regards!

Reply:
True! Yes, moved the ifdef outside the function, thanks!